gap/oper.gi

```diff
@@ -1005,13 +1005,13 @@ function(digraph, edge)
 end);
 
 InstallMethod(IsDigraphEdge, "for a digraph, pos int, pos int",
-[IsDigraph, IsPosInt, IsPosInt],
+[IsDigraph, IsInt, IsInt],
 function(digraph, u, v)
   local n;
 
   n := DigraphNrVertices(digraph);
 
-  if u > n or v > n then
+  if u > n or v > n or u < 0 or v < 0 then
```

Comment:
Why do we want to allow `0`? Vertices are currently restricted to `[1..n]`

Reply:
Double sorry my bad. BTW the reason for this change was that I want to call `IsDigraphEdge` in the C code, and there, sometimes GAP integers don't seem belong to `IsPosInt` even though they are positive and would satisfy `IsPosInt` in GAP.